tests/onnx/test_nncf_graph_builder.py

```diff
@@ -0,0 +1,32 @@
+import os
+import pytest
+
+from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
+
+from tests.onnx.models import LinearModel
+from tests.onnx.models import MultiInputOutputModel
+
+import networkx as nx
+
+TEST_MODELS = [LinearModel(), MultiInputOutputModel()]
```

Comment:
I'd use model_creator function, instead of instantiated model itself. 
Currently, all tests in the pre-commit scope will fail on test collection if there's error on creation of some model. 
With model_creator tests will be collected successfully and only one test case will fail, all others won't be affected.

Reply:
Good remark!

Reply:
Implement this idea